src/test/java/converter/StringToUserConverterTest.java

```diff
@@ -0,0 +1,94 @@
+package converter;
+
+import model.User;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class StringToUserConverterTest {
+    final StringToUserConverter stringToUserConverter = new StringToUserConverter();
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertEmptyString() {
+        stringToUserConverter.convert(" ");
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertEmptyFields() {
+        String actual = "" + " " + "" + " " + "" + "";
+        stringToUserConverter.convert(actual);
+    }
+
+    @Test(expected = IllegalArgumentException.class)
+    public void testConvertNullString() {
+        stringToUserConverter.convert(null);
+    }
+
+
+    @Test
+    public void testConvertNameWithUnderSlash() {
```

Comment:
`-` and `_` are the same cases
but I'd recommend you to make test for `-` as it is more likely